ws_api/wealthsimple_api.py

```
@@ -690,6 +690,28 @@ def get_security_market_data(self, security_id: str, use_cache: bool = True):
 
         return value
 
+    def get_security_dividend_details(self, security_id: str, currency: str = None):
```

Ruff (0.15.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)